### PR TITLE
Vision GAPIC client library

### PR DESCRIPTION
**Pull request by dizcology**
Updating the sample codes to use the GAPIC client library.

Merge only after the client library has been released.

### CODE REVIEW

File: vision/cloud-client/detect/detect.py [hunk 1]
```diff
@@ -32,60 +32,77 @@
...
+
+# Names of likelihood from google.cloud.vision.enums
+likelihood_name = ('UNKNOWN', 'VERY_UNLIKELY', 'UNLIKELY', 'POSSIBLE',
+                   'LIKELY', 'VERY_LIKELY')
 
 
 def detect_faces(path):
     """Detects faces in an image."""
-    vision_client = vision.Client()
+    # Names of likelihood from google.cloud.vision.enums
```
Comment (jonparrott) [inline vision/cloud-client/detect/detect.py]: Why have this global and local? pick one?

**Comment by jonparrott** [comment]
This looks good, but you need to resolve the conflicts first.

File: vision/cloud-client/detect/detect.py (hunk 1 above)
Comment (dizcology) [inline vision/cloud-client/detect/detect.py]: updated.  keeping the local ones.

**Comment by dizcology** [comment]
rebased and incremented client library version.

merge only after the client library is published and tested.

**Comment by dizcology** [comment]
client library was published and tested against these updated samples.

**Pull request merged**